### ISSUE TITLE
[Stacked Themes] The plugin `flutter_statusbarcolor_ns` uses a deprecated version of the Android embedding

### ISSUE THREAD

**Issue by ajnaf** [Stacked-Org/stacked#501]
The plugin `flutter_statusbarcolor_ns` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
i was going to open an issue to the author but he closed the issue section.
fyi this happened in the latest flutter update 2.5

**Comment by dzab5** [comment]
having the same issue!

**Comment by azazadev** [comment]
same issue, should we open ticket on flutter_statusbarcolor_ns  side ?

**Comment by dzab5** [comment]
yes go ahead

**Comment by azazadev** [comment]
> yes go ahead

not able to create ticket on git repo flutter_statusbarcolor_ns

**Comment by anuragsoni300** [comment]
having same issue

**Comment by sssergei** [comment]
have same issue :( after update to last version

**Comment by azazadev** [comment]
any news ? it is possible to copy the flutter_statusbarcolor_ns package source file to stacked package ? it is only 1 file with 74 line 
https://github.com/Sameerkash/flutter_statusbarcolor/tree/master/lib

**Comment by sssergei** [comment]
I find where it is in my code. I am using the stacked_themes: ^0.3.2

**Comment by emri99** [comment]
Fix seems to be done on a fork of flutter_statusbarcolor: https://github.com/takeshishimada/flutter_statusbarcolor/tree/debug

Unfortunately, overriding the dependency in pubspec.yaml **doesn't seem to work any better**

pubspec.yaml
```yaml
dependency_overrides:
  flutter_statusbarcolor_ns:
    git:
      url: https://github.com/takeshishimada/flutter_statusbarcolor
      ref: debug
```

Still having the error on `pub get` 

```
Running "flutter pub get" in myapp...  
Warning: You are using these overridden dependencies:
! flutter_statusbarcolor_ns 0.3.0-nullsafety from git https://github.com/takeshishimada/flutter_statusbarcolor at 241993

Running "flutter pub get" in myapp...                   1 424ms
The plugins `flutter_statusbarcolor_ns` use a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if these plugins support the Android V2 embedding. Otherwise, consider removing them since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
``` 

Also tried to add in dependency section, but it raise a conflict as expected.

Hope someone succeeds where I fail and get us in touch ;)

**Comment by FilledStacks** [comment]
@hurbes can we look at this?

**Comment by sqrg** [comment]
Same issue here. Is there any workaround?

**Comment by emri99** [comment]
Wait for 0.3.7 to be released or maybe use git  dependency format for this package in pubspec to get the fix from above merged PR

**Comment by myklll** [comment]
The new version of flutter_statusbarcolor_ns is out and fixes this. @FilledStacks @hurbes

**Comment by luisredondo** [comment]
> The new version of flutter_statusbarcolor_ns is out and fixes this. @FilledStacks @hurbes

Yes. So basically to anyone still struggling with this, just add this to your pubspec.yaml

```
dependency_overrides:
    flutter_statusbarcolor_ns: ^0.4.0
```

**Comment by luisredondo** [comment]
> > The new version of flutter_statusbarcolor_ns is out and fixes this. @FilledStacks @hurbes
> 
> Yes. So basically to anyone still struggling with this, just add this to your pubspect.yaml
> 
> ```
dependency_overrides: 
    flutter_statusbarcolor_ns: ^0.4.0
````

**Comment by Edamijueda** [comment]
> > The new version of flutter_statusbarcolor_ns is out and fixes this. @FilledStacks @hurbes
> 
> Yes. So basically to anyone still struggling with this, just add this to your pubspec.yaml
> 
> ```
> dependency_overrides:
>     flutter_statusbarcolor_ns: ^0.4.0
> ```

Thank you. This filled my issue. But at the same time displayed a warning message:

Warning: You are using these overridden dependencies:
! flutter_statusbarcolor_ns 0.4.0

I guess this warning was displayed to notify me of my action

**Comment by azazadev** [comment]
any news to release this please ? and also moving provider to latest 6.0.1

**Comment by surveshoeb** [comment]
@azazadev upgrade your `stacked_themes` package. If you are still facing an issue upgrade then upgrade your kotlin version in build.gradle `ext.kotlin_version = '1.4.20'`

**Comment by FilledStacks** [comment]
This will be fixed soon. Thanks for being patient with me 😄

**Comment by Eimen2018** [comment]
Fixed on version: >= 0.3.7
